apps/fluent-tester/src/FluentTester/TestComponents/Text/PressableUsage.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import { Alert, Linking, Platform, View } from 'react-native';
+import { Stack } from '@fluentui-react-native/stack';
+import { Text } from '@fluentui/react-native';
+import { stackStyle } from '../Common/styles';
+import { IKeyboardEvent, IHandledKeyboardEvent } from '@office-iss/react-native-win32';
+
+export const PressableUsage: React.FunctionComponent<{}> = () => {
+  if (Platform.OS == ('win32' as any)) {
```

Comment:
Not strictly necessary to platform guard is it? keyDown just won't do anything on the other platform, worth adding a comment perhaps, maybe even a platform check in dev mode inside the Text component itself that logs a warning or error.

Reply:
I know that @HeyImChris recently added keyboarding to react-native-macos so maybe it does? He would be better to comment though

Reply:
I see that, looks like it works slightly differently than it does for win32 though. I can check to see if it causes any issues.